NICE.Identity.Management/ClientApp/src/setupTests.js

```diff
@@ -3,4 +3,13 @@ import Adapter from "enzyme-adapter-react-16";
 
 configure({ adapter: new Adapter() });
 
-export default undefined;
+beforeEach(() => {
```

Comment:
Is it worth putting a comment with the link to the tweet as to where this came from so someone can understand the rationale?